tests/test_group.py

```diff
@@ -102,3 +102,27 @@ def test_github_non_nda_user_cannot_create_access_group(self, base_url, selenium
         groups_page = home_page.header.click_groups_menu_item()
         create_group_page = groups_page.click_create_group_main_button()
         assert not create_group_page.is_access_group_present
+
+    @pytest.mark.credentials
+    def test_cannot_invite_github_non_nda_user_to_access_group(self, base_url, selenium, ldap, github_non_nda_user):
+        home_page = Home(selenium, base_url).open()
+        home_page.login_with_ldap(ldap['email'], ldap['password'], ldap['secret'])
+
+        # Create a new group of type "Reviewed" and access "Access group"
+        group_name = 'moz-group-access-{0}'.format(uuid.uuid4())
+        groups_page = home_page.header.click_groups_menu_item()
+        group = groups_page.create_group(group_name, "Reviewed", "Access Group")
+        assert "Reviewed" in group.access.group_type.current_group_type
+        assert "Access Group" in group.access.access_form.selected_access_type
+
+        # Try to invite a non nda user to the access group
+        invite_form = group.invitations.invite
+        invites = len(invite_form.invites_list)
+        invite_form.invite_new_member(github_non_nda_user['username'])
+        assert len(invite_form.invites_list) == invites
+
+        # Try to add a non nda user as curator of the access group
+        curators_form = group.access.curators_form
+        curators = len(curators_form.curators_list)
+        curators_form.add_curator(github_non_nda_user['username'])
+        assert len(curators_form.curators_list) == curators
```

Comment:
Should invite/add curator be separate tests?

Reply:
I will split the test in 2 separate tests.